src/doc/trpl/crates-and-modules.md

```diff
@@ -430,7 +388,7 @@ fn main() {
 }
 ```
 
-But it is not idiomatic. This is significantly more likely to introducing a
+This is more likely to introduce a
```

Comment:
I felt it was enough to simply mention the naming error issue. That it may not be idiomatic is secondary.